rust/datafusion/src/lib.rs

```diff
@@ -52,6 +52,7 @@
     clippy::useless_format,
     clippy::zero_prefixed_literal
 )]
+#![feature(box_patterns)]
```

Comment:
Box patterns are unstable and only available in nightly Rust, which we are trying to move away from, so we should add this.

Reply:
Ok I found a way around it.